tests/service/PatternMatchCompilationTests.fs

```diff
@@ -1260,4 +1260,20 @@ let f : obj -> _ =
     assertHasSymbolUsages ["i"] checkResults
     dumpDiagnostics checkResults |> shouldEqual [
         "(5,6--5,18): Feature 'non-variable patterns to the right of 'as' patterns' is not available in F# 5.0. Please use language version 6.0 or greater."
-    ]
\ No newline at end of file
+    ]
+
+[<Test>]
+#if !NETCOREAPP
+[<Ignore("These tests weren't running on desktop and this test fails")>]
```

Comment:
Random question - won't it work actually? I think most of the tests in this suite will pass on desktop now.
(Not asking to reenable all the tests here - wondering if this is copypaste or deliberate)

Reply:
It didn't work previously, and nowadays I just copy it as a template. 🙂